recipes-graphics/flutter-engine/flutter-engine_git.bb

```diff
@@ -133,17 +133,9 @@ do_configure[depends] += "depot-tools-native:do_populate_sysroot"
 
 do_compile() {
 
-    FLUTTER_RUNTIME_MODES="\
-        ${@d.getVarFlags("PACKAGECONFIG").get('debug')} \
-        ${@d.getVarFlags("PACKAGECONFIG").get('profile')} \
-        ${@d.getVarFlags("PACKAGECONFIG").get('release')} \
-        ${@d.getVarFlags("PACKAGECONFIG").get('jit_release')}"
+    FLUTTER_RUNTIME_MODES="$(echo "${PACKAGECONFIG_CONFARGS}" | grep -o -P -e '(?<=--runtime-mode )(\w+)')"
```

Comment:
Consider using the `bb.utils.filter` function instead:
`FLUTTER_RUNTIME_MODES="${@bb.utils.filter('PACKAGECONFIG', 'debug profile release jit_release', d)}"`
This ensures that only those 4 runtime modes can end up in `FLUTTER_RUNTIME_MODES` (if they are set in `PACKAGECONFIG`).